action.yaml

```diff
@@ -7,15 +7,16 @@ inputs:
   container-sign-kms-key-arn:
     description: "The secret with the ARN of the key to sign container images e.g. secrets.CONTAINER_SIGN_KMS_KEY"
     required: false
-    default: "none"
   template-file:
     description: "The name of the CF template for the application. This defaults to template.yaml"
     required: false
     default: template.yaml
   working-directory:
-    description: "The working directory containing the app"
+    description: "Deprecated - use sam-base-directory instead"
```

Comment:
is this a breaking change and will we need a new major release? Can you update the readme also :)